packages/orchestrator/lib/routes/v1/tasks/taskId/getOutput.ts

```diff
@@ -49,8 +50,13 @@ const handler = (scheduler: Scheduler, eventEmitter: EventEmitter) => {
                 respond(res);
             }
         };
-        const onCompletion = (completedTask: Task) => {
-            cleanupAndRespond((res) => res.status(200).json({ state: completedTask.state, output: completedTask.output }));
+        const onCompletion = async (taskId: Task['id']) => {
+            const completedTask = await scheduler.get({ taskId });
+            if (completedTask.isErr()) {
+                cleanupAndRespond((res) => res.status(404).json({ error: { code: 'task_not_found', message: completedTask.error.message } }));
+                return;
+            }
+            cleanupAndRespond((res) => res.status(200).json({ state: completedTask.value.state, output: completedTask.value.output }));
```

Comment:
here is the drawback of using pg. Before we could pass the entire task as the argument of the event when everything was happening locally, but pg has a 8kb limit for the payload of NOTIFY. We therefore only passing the taskId and retrieve the state and output by querying the db